File: cesar.py
def converter(s, y):
    return ''.join(c if c == ' ' else chr(ord(c) + y) for c in s)

def deconverter(s, y):
    return ''.join(c if c == ' ' else chr(ord(c) - y) for c in s)

i = int(input("Input the shift number: "))
m = input("Input the message: ")

hid = converter(m, i)
dehid = deconverter(hid, i)

print("Original  | " + m)
print("Encoded   | " + hid)
print("Decoded   | " + dehid)

File: cesar.java
import java.util.Scanner;

public class cesar {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.print("Input the shift number: ");
        int i = s.nextInt();
        s.nextLine();
        System.out.print("Input the message: ");
        String m = s.nextLine();

        String hid = converter(m, i);
        String dehid = deconverter(hid, i);
        System.out.println("Original  | " + m);
        System.out.println("Encoded   | " + hid);
        System.out.println("Decoded   | " + dehid);
    }

    public static String converter(String s, int y) {
        String ne = "";
        for (int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            ne += (c == ' ') ? c : (char)(c + y);
        }
        return ne;
    }

    public static String deconverter(String s, int y) {
        String ne = "";
        for (int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            ne += (c == ' ') ? c : (char)(c - y);
        }
        return ne;
    }
}
